capybara-ui.gemspec

```diff
@@ -22,12 +22,14 @@ Gem::Specification.new do |s|
   s.add_dependency 'capybara', '~> 3.0'
 
   s.add_development_dependency 'rspec-given', '~> 3.5.0'
-  s.add_development_dependency 'capybara-webkit', '~> 1.7.0'
-  s.add_development_dependency 'poltergeist', '~> 1.17.0'
+  s.add_development_dependency 'selenium-webdriver'
+  s.add_development_dependency 'webdrivers'
+  s.add_development_dependency 'cuprite'
   s.add_development_dependency 'cucumber', '~> 1.3.0'
   s.add_development_dependency 'pry'
   s.add_development_dependency 'pry-nav'
+  s.add_development_dependency 'puma'
   s.add_development_dependency 'sinatra'
```

Comment:
Can we drop sinatra or is it still used?  I noticed commits in a git blame that talked about transitioning from sinatra to rails in the tests?

Reply:
Looks like Sinatra is still being used in the RSpec tests.

https://github.com/mojotech/capybara-ui/blob/a85620e9733f18eadea69cd23741b9df6baf58a5/spec/spec_helper.rb#L16

Reply:
👍